docs/configure/content-set/navigation.md

```diff
@@ -73,6 +73,10 @@ toc:
   - file: index.md
 ```
 
+:::{important}
+The first file must always be named `index.md`.
+:::
```

Comment:
This is actually not true, the first listed file is the `index`. 

We should avoid having a ton of `index.md` files to aid with contributing IMO.

Reply:
Yay, I'm glad it's not a requirement. I'll close this PR.